scan/src/components/modal/submitTx/ReinvestMsg.re

```diff
@@ -0,0 +1,73 @@
+module Styles = {
+  open Css;
+
+  let container = style([paddingBottom(`px(24))]);
+};
+
+[@react.component]
+let make = (~validator, ~amount, ~setMsgsOpt) => {
+  let validatorInfoSub = ValidatorSub.get(validator);
+
+  React.useEffect0(_ => {
+    let msgsOpt = {
+      let%Opt amountValue = Some(amount);
+      Some([|
+        TxCreator.WithdrawReward(validator),
+        TxCreator.Delegate(
+          validator,
+          {amount: amountValue |> Js.Float.toFixedWithPrecision(~digits=0), denom: "uband"},
```

Comment:
I guess this function will automatically round up if the remainder is more than 5. So, I think you have to use `amountValue->Js.Math.floor_float->Js.Float.toString` to do it.